config/cucumber.yml

```diff
@@ -1,2 +1,3 @@
-default: --format pretty  --profile semaphoreci
+default: --format pretty  --profile semaphoreci --profile semaphoreci
+semaphoreci: --format json --out=../cucumber_report.json
 semaphoreci: --format json --out=../cucumber_report.json
```

Comment:
Very good question. 

I was also wondering the same thing several days ago. The cucumber_injector adds a new line whenever I execute my tests.

To combat this, I have this file to `.gitignore` (because it is not really used for this project).

Hopefully, with a bit of tweeks here and there, I will be able to run cucumber_booster without dirtying the current working directory. Unfortunately, we are not there yet, and it will require a bit more effort.

Thanks for remaining me about this. I'll make sure to document or open an issue.